src/sentry/features/temporary.py

```diff
@@ -176,6 +176,7 @@ def register_temporary_features(manager: FeatureManager):
     # Adds the ttid & ttfd vitals to the frontend
     manager.add("organizations:mobile-vitals", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
     # Enables higher limit for alert rules
+    manager.add("organizations:more-fast-alerts", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
```

Comment:
TODO - investigate how to add this feature flag / target only a single org (kinda like the `more-slow` handler) or if i need to handle it in options automator.